.github/scripts/ensure-tag.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Isn't there a GitHub action that is similar to what this script is doing?

Reply:
Probably. But I prefer not adding another dependency (that can break, or be compromised) for something so simple.

I will look into it if it doesn't work.